repository.yaml

```
@@ -1,4 +1,4 @@
 # https://developers.home-assistant.io/docs/add-ons/repository#repository-configuration
 name: Asterisk Add-on
-url: "https://github.com/TECH7Fox/Asterisk-add-on"
-maintainer: Jordy Kuhne <jordy.kuhn@gmail.com>
\ No newline at end of file
+url: "https://github.com/TECH7Fox/asterisk-hass-addons"
+maintainer: Jordy Kuhne <jordy.kuhn@gmail.com>
```

I now see that I missed a 'e'.

```suggestion
maintainer: Jordy Kuhne <jordy.kuhne@gmail.com>
```